### PR TITLE
Fixes #305 - Added: Shared Draft API

### LINKED ISSUE #305: Shared draft API is missing

### PR DESCRIPTION
**Pull request by ralf401**

### CODE REVIEW

File: api/ticket/shared_drafts.rst [hunk 1]
```diff
@@ -0,0 +1,1838 @@
...
+
+   {
+      "shared_draft_id": 2,
+      "assets": {
+         "TicketSharedDraftZoom": {
+            "2": {
+               "id": 2,
+               "ticket_id": 58,
+               "new_article": {
+                  "body": "<div>Some text for a shared draft. <br></div><div><br></div><div>Some inline image:<img tabindex=\"0\" style=\"width: 1000px; max-width: 100%;\" src=\"data:image/jpeg;base64,/9j/4AAQSkZJRg[...]
```
Comment (mgruner) [inline api/ticket/shared_drafts.rst:26]: Cutting off withouth the final `",` confuses the syntax highlighting in the final output. Please correct this (`[...]",`)

File: api/ticket/shared_drafts.rst [hunk 2]
```diff
@@ -0,0 +1,1838 @@
+Shared drafts
```
Comment (mgruner) [inline api/ticket/shared_drafts.rst:1]: Title Case: `Shared Drafts`?

File: api/ticket/shared_drafts.rst [hunk 3]
```diff
@@ -0,0 +1,1838 @@
...
+*************
+
+Show
```
Comment (mgruner) [inline api/ticket/shared_drafts.rst:4]: There is one additional currently undocumented route, `import_attachments`. Maybe you can check with @mantas if we should document this as well, and how it works.

**Review by mgruner — APPROVED** [review]
Looks nice, minor suggestions to consider.

File: api/ticket/shared_drafts.rst (hunk 2 above)
Comment (ralf401) [reply api/ticket/shared_drafts.rst:1]: Some day I will learn it ;-)

File: api/ticket/shared_drafts.rst (hunk 3 above)
Comment (ralf401) [reply api/ticket/shared_drafts.rst:4]: I got in touch with him and we considered it as not important. He states that there is probably just one use case which is not very feasible (user opens ticket and draft > copies the form ID > calls the API with this ID to import an attachment).

But thanks for your review and keeping your eyes open anyway :-)

**Pull request merged**